core/bdo.ttl

```diff
@@ -299,8 +299,8 @@ In general include the use Copyright Page Title when it differs from the bibliog
 		;
 	adm:userTooltip "Title appearing on the copyright page of a Work, if present."@en ;
 	rdfs:seeAlso
-		"http://cip.loc.gov/cipman/"^^xsd:anyURI ,
```

Comment:
that's an interesting change... the anyURI business was on purpose, with the goal to tell RDF systems not to consider these as RDF resources that might have predicates, etc.

what's your take on that?

Reply:
You're right, that was a mistake on my part. I'll change it. BTW that's a dead link, so it should probably be removed. Let me know what you want to do.

Reply:
indeed... I looked on the wayback machine and I'm not sure it was that relevant in the first place so let's remove it. Thanks!

Reply:
I asked my colleagues at Semantic Arts about the difference between the use of a typed string vs a URI resource in RDF, and the consensus is there is no reason to use the typed string in RDF generally.

There are instances where SHACL requires the use of the typed string. E.g., 

The values of `sh:namespace` are literals of datatype `xsd:anyURI`.

```
	sh:declare [
		sh:prefix "schema" ;
		sh:namespace "http://schema.org/"^^xsd:anyURI ;
	] .
```

So I've left these changes as they are, but if you have strong feelings in the other direction I can pull them out.

Reply:
ah, that's interesting, thanks a lot for this clarification!